### PR TITLE
Add October 2019 slide deck

### PR DESCRIPTION
**Pull request by WyriHaximus**
* Updated @henriquemoody photo
* Added Hacktobefest
* Added serverless workshop
* Added dumpsterfire

### CODE REVIEW

File: 2019-October.html [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+                <section>
+                    <h1 class="slide fancy">Conferences and Events</h1>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Hacktoberfest</h1>
+                    <img width="600" src="img/hacktoberfest.svg">
+                    <h4>October</h4>
+                    <h4>::1</h4>
```
Comment (carusogabriel) [inline 2019-October.html]: Stands for `localhost`? 🤓

File: 2019-October.html [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+                    <h1 class="slide fancy">Serverless PHP workshop</h1>
+                    <img width="600" src="img/serverless-workshop.jpg">
+                    <h4>December 2th-3th</h4>
+                    <h4>Amsterdam</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Developer survey</h1>
+                    <img width="600" src="img/DE17-tweet.png">
+                    <h4>Until end of July</h4>
```
Comment (carusogabriel) [inline 2019-October.html]: Can we remove this one?

File: 2019-October.html [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+                </section>
+
+                <section>
+                    <h1 class="fancy">Rating</h1>
+
+                    <div class="two-columns">
+                        <div>
+                            <img class="meetup" src="img/meetup-large.png" alt="Meetup">
+                            <h2 class="fancy">The Venue</h2>
+                        </div>
```
Comment (carusogabriel) [inline 2019-October.html]: Bye bye meetup, please remove it.

File: 2019-October.html [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+        <!--[if lt IE 9]>
+        <script src="lib/js/html5shiv.js"></script>
+        <![endif]-->
+
+        <script>
+            slideData = {
+                "text": {
+                    "date": "October 2019",
+                    "speaker-name": "Henrique Moody",
+                    "joind-in": "https://joind.in/talk/06a20",
```
Comment (carusogabriel) [inline 2019-October.html]: We need a new link.

File: 2019-October.html (hunk 1 above)
Comment (WyriHaximus) [reply 2019-October.html]: Yes the `IPv6` `localhost` :nerd_face:

File: 2019-October.html (hunk 4 above)
Comment (WyriHaximus) [reply 2019-October.html]: Yes, couldn't find it so quickly earlier

**Pull request merged**